app/helpers/filters_helper.rb

```diff
@@ -19,4 +19,8 @@ def search_path(type)
   def auto_complete_search_path
     '/auto_complete_search'
   end
+
+  def get_resource_type(resource_type)
+    resource_type == _('(Miscellaneous)') ? nil : resource_type
```

Comment:
Comparing translated resources feels fragile. Especially when one can be lazily translated and the other isn't.

Reply:
changed to `_(resource_type) == _('(Miscellaneous)')`, will that do?

Reply:
Can we ensure resource type is always untranslated?

Reply:
Yes, fixed, thanks